packages/swap/src/routes/swap.ts

```diff
@@ -132,6 +133,20 @@ router.get(
       );
     }
 
+    let depositChannelExpiryTime: Date | null = null;
+    if (swapDepositChannel && !swapDepositChannel.isExpired) {
+      const chainInfo = await prisma.chainTracking.findFirst({
+        where: {
+          chain: swapDepositChannel?.srcChain,
+        },
+      });
+      depositChannelExpiryTime = calculateExpiryTime({
+        chain: swapDepositChannel.srcChain,
+        startBlock: chainInfo?.height,
```

Comment:
can we not simply use the `swapDepositChannel.issuedBlock` instead of `chainInfo.height` and always return a timestamp with the expected expiration time?

Reply:
issued block is not the issued block in source chain. Expiry on the other hand happens based on the source chain's block

Reply:
Ah, now it makes sense.